packages/nodejs/src/clientV2.ts

```diff
@@ -0,0 +1,167 @@
+    let returnIdentifier: any = {};
+
+    if (accountIdentifier instanceof AccountAddressLocal) {
+        const decodedAddress = new Uint8Array(accountIdentifier.decodedAddress);
+        returnIdentifier.oneofKind = "address"
+        returnIdentifier.address = {value: decodedAddress}
+    } else if (accountIdentifier instanceof CredentialRegistrationIdLocal) {
+        const credId = Buffer.from(accountIdentifier.credId, 'hex');
+        returnIdentifier.oneofKind = "credId"
+        returnIdentifier.credId = {value: credId}
+    } else {
+        returnIdentifier.oneofKind = "accountIndex"
+        returnIdentifier.accountIndex = {value: accountIdentifier};
+    }
+
+    return {accountIdentifierInput: returnIdentifier}
+}
+
+/**
+ * A concordium-node specific gRPC client wrapper.
+ *
+ * @example
+ * import ConcordiumNodeClient from "..."
+ * const client = new ConcordiumNodeClient('127.0.0.1', 20000, credentials, metadata, 15000);
+ */
+export default class ConcordiumNodeClient {
+    client: QueriesClient;
+
+    metadata: Metadata;
+
+    address: string;
+
+    port: number;
+
+    timeout: number;
+
+    /**
+     * Initialize a gRPC client for a specific concordium node.
+     * @param address the ip address of the node, e.g. 127.0.0.1
+     * @param port the port to use when econnecting to the node
+     * @param credentials credentials to use to connect to the node
+     * @param timeout milliseconds to wait before timing out
+     * @param options optional options for the P2PClient
+     */
+    constructor(
+        address: string,
+        port: number,
+        credentials: ChannelCredentials,
+        metadata: Metadata,
+        timeout: number,
+        options?: Record<string, unknown>
+    ) {
+        if (timeout < 0 || !Number.isSafeInteger(timeout)) {
+            throw new Error(
+                'The timeout must be a positive integer, but was: ' + timeout
+            );
+        }
+
+        const grpcTransport = new GrpcTransport({
+            host: `${address}:${port}`,
+            channelCredentials: credentials,
+            options: options,
+        });
+
+        this.address = address;
+        this.port = port;
+        this.timeout = timeout;
+        this.metadata = metadata;
+        this.client = new QueriesClient(grpcTransport);
+    }
+
+    /**
+     * Retrieves the next account nonce for the given account. The account nonce is
+     * used in all account transactions as part of their header.
+     * @param accountAddress base58 account address to get the next account nonce for
+     * @returns the next account nonce, and a boolean indicating if the nonce is reliable
+     */
+    async getNextAccountSequenceNumber(
+        accountAddress: AccountAddressLocal
+    ): Promise<NextAccountSequenceNumber> {
+        const address: AccountAddress = {
+            value: new Uint8Array(accountAddress.decodedAddress)
```

Comment:
```suggestion
            value: accountAddress.decodedAddress
```
I assume that you don't need to wrap the buffer in a new Uint8Array. (Wasn't this only a problem with the other library?)